code/TCP_Srv_Base.cpp

```diff
@@ -195,7 +202,37 @@ void TCP_Srv_Base::endOfThread()
         if (mCnt == 0) 
         {
             mNum = 0;
-            TRACE("--- no clients ---")
+            Trace() << "--- no clients ---" << endl;
         }
+        displayThreads();
     }
-}
\ No newline at end of file
+}
+
+void TCP_Srv_Base::displayThreads() const
+{
+    if constexpr (not Trace::isOn)
```

Comment:
maybe introduce a reversed Trace class

Reply:
- called it Whisper
- extern objekts now
  - trace
  - whisper